examples/graphql_examples/tests/src/Kernel/Schema/ExampleSchemaTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Drupal\Tests\graphql_example\Kernel;
+
+use Drupal\node\Entity\Node;
+use Drupal\node\Entity\NodeType;
+use Drupal\Tests\graphql\Kernel\GraphQLTestBase;
+use Drupal\user\Entity\User;
+
+class ExampleSchemaTest extends GraphQLTestBase {
```

Comment:
Can you add a class comment here that this test executes against the graphql_examples schema?